csharp/src/Google.Protobuf.Test/WellKnownTypes/WrappersTest.cs

```diff
@@ -417,5 +417,12 @@ public void ClearWithReflection()
             TestWellKnownTypes.Descriptor.Fields[TestWellKnownTypes.StringFieldFieldNumber].Accessor.Clear(message);
             Assert.IsNull(message.StringField);
         }
+
+        [Test]
+        public void GivenBoolValueWhenPerformingRoundTripEncodingViaJsonThenShouldNotExpectObjectAtTopLevel()
+        {
+            var value = new BoolValue { Value = true };
+            Assert.That(value, Is.EqualTo(JsonParser.Default.Parse<BoolValue>(JsonFormatter.Default.Format(value))));
```

Comment:
Could you rewrite this to use Assert.AreEqual, to be consistent with the rest of the tests?